api/household/householdModel.js

```diff
@@ -0,0 +1,44 @@
+const knex = require('../../data/db-config');
+
+const create = async (households) => {
+  let newHouseholdId;
+  try {
+    await knex.transaction(async (trx) => {
```

Comment:
It doesn't harm anything in this case, but a transaction isn't necessary here. 

It's only needed when you need to do insertions into several different tables at once.  

How it works, is that if something goes wrong when inserting into one of the tables, all the changes so far get rolled back. In that way, you don't have a situation where you have random pieces of data added to your tables.

Reply:
Good to know!  Thank you